cmd/info.js

```diff
@@ -40,11 +41,20 @@ const output = outputter('info', {
   keys,
   info,
   changelog,
-  error: ({ code, stack }) => `Info Error (code: ${code || 'none'}) ${stack}`
+  error: (err, info, ipc) => {
```

Comment:
I know this exists already but I think we should actually throw instead of using error tag

Reply:
this is used elsewhere also - I think separate to this we should use try catch and not error tag for ops

Reply:
but not change needed here